src/sentry/web/frontend/error_page_embed.py

```diff
@@ -208,7 +209,7 @@ def dispatch(self, request):
         )
 
         context = {
-            "endpoint": mark_safe("*/" + json.dumps(request.build_absolute_uri()) + ";/*"),
+            "endpoint": settings.SENTRY_OPTIONS["system.url-prefix"],
```

Comment:
I think you need the surrounding `*/` and `;/*`. The template uses this variable in the following way:

```
  var endpoint = /*{{ endpoint }}*/'';
```
Which won't work with the current value.

Reply:
Good now?

PS: I have no idea about how to test this.

Reply:
This endpoint serves the user-feedback dialog. You can test it by setting up the browser-sdk https://docs.sentry.io/enriching-error-data/user-feedback/?platform=browser